tests/orquestra/quantum/measurement_test.py

```diff
@@ -866,3 +862,20 @@ def test_check_sample_elimination(
         for sample in correct_samples:
             bitstring = tuple([int(measurement_value) for measurement_value in sample])
             assert correct_samples[sample] <= bitstring_counts[bitstring]
+
+
+@pytest.mark.parametrize(
+    ("counts", "active_qubits", "new_count"),
+    [
+        (
+            {"001": 100, "010": 101, "011": 7},
+            [0, 1, 2],
+            {"001": 100, "010": 101, "011": 7},
+        ),
+        ({"001": 100, "010": 101, "011": 7}, [0, 1, 3], ValueError),
+    ],
+)
+def test_if_subdistribution_works(
+    counts, active_qubits,
+):
+    assert subdistribution(counts, active_qubits) == new_count
```

Comment:
Hi, @mstechly I am not actaully sure how to import ```subdistribution```.  I found this site that says that the == should work for simple dictionaries. 
This is the site: https://pythonguides.com/check-if-two-dictionaries-are-equal-in-python/#:~:text=Check%20if%20two%20nested%20dictionaries%20are%20equal%20in%20Python,-In%20this%20example&text=To%20do%20this%20task%2C%20we,dictionaries%20are%20equal%20or%20not.

Reply:
I think I figure out how to import

Reply:
the new_count still says not define for some reason ```assert _measurement_outcome_distribution.subdistribution(counts, active_qubits) == new_count```

Reply:
Hey, so because you currently have written subdistribution as a standalone function, it must be imported in order to test it. It would be better to make `subdistribution` a method of the `MeasurementOutcomeDistribution` class. Then it wouldn't have to be imported as it would come alone with the import of `MeasurementOutcomeDistribution`.

Reply:
@AthenaCaesura Oh I understand, thank you